staff/manuel-barzi/liveshare/app/src/logic/modifyPost.js

```diff
@@ -0,0 +1,28 @@
+function removePost(postId, text) {
```

Comment:
Esta función se debería llamar modifyPost. Un saludo
```suggestion
function modifyPost(postId, text) {
```